cumulus/parachains/runtimes/assets/asset-hub-westend/src/lib.rs

```diff
@@ -1451,13 +1645,23 @@ mod benches {
 	frame_benchmarking::define_benchmarks!(
 		[frame_system, SystemBench::<Runtime>]
 		[frame_system_extensions, SystemExtensionsBench::<Runtime>]
+		[pallet_asset_conversion_ops, AssetConversionMigration]
+		[pallet_asset_rate, AssetRate]
 		[pallet_assets, Local]
 		[pallet_assets, Foreign]
 		[pallet_assets, Pool]
 		[pallet_asset_conversion, AssetConversion]
 		[pallet_asset_rewards, AssetRewards]
 		[pallet_asset_conversion_tx_payment, AssetTxPayment]
+		[pallet_bags_list, VoterList]
 		[pallet_balances, Balances]
+		[pallet_conviction_voting, ConvictionVoting]
+		// Temporarily disabled due to https://github.com/paritytech/polkadot-sdk/issues/7714
```

Comment:
We can leave this disabled for Westend but would need it for Kusama. cc @kianenigma